diesel_codegen/src/schema_inference.rs

```diff
@@ -1,4 +1,5 @@
 use syn;
+// use syntex_syntax::symbol::keywords;
```

Comment:
Yes, I was trying to use syntax to work around some issues but don't need it right now. I'll kill the dependency if it's not needed (which it probably won't be) before any merge.